### PR TITLE
291 ensure coding standards

### LINKED ISSUE #291: Ensure Coding Standards Are Implemented
As a developer, I'd like to have all the coding standards implemented, in order to have clean code.

---

### :thought_balloon: &nbsp; Hints & Thoughts
Should be one of the last user stories to be done. 

### :question: &nbsp; Additional Questions to Answer / Consider

---

## :white_check_mark: &nbsp; Conditions of satisfaction

- [ ] Should make sure that all the rules determined in our coding standards are also used in throughout the code (Backend, Frontend, Local Client)

### PR DESCRIPTION
**Pull request by WolfgangDaniel**
closes #291 

Please also review bptlab/ark_automate_local#13 

## Changes
Went over all the files and fixed all the coding styles violations that we could find.

## Checklist before merge

**Developer's responsibilities**
* [ ] **Assign** one or two developers
* [ ] **Change code** if reviewer(s) has/have requested it
* [ ] **Pull request build** has passed
* [ ] **tested locally** (in at least chrome & firefox if frontend)
* [ ] updated the **documentation**
* [ ] added **tests** where necessary

### CODE REVIEW

**Review by SanJSp — APPROVED** [review]
Thank you for your work 😊 I've got two small comments that are up to you to fix. First I'd like to make a point to leave the "verify if really in DB" comments or to adapt the variables use when verifying to show, that it is actually about verification in the second call of each test. Next, I've added some small comments to places where things popped up. Nothing too serious, therefore I approve with the changes in general.

Lastly, I've run prettier over all files and it found no serious corrections. Therefore I think everything is fine :)

File: server/api/routes/robots/robots.test.js [hunk 1]
```diff
@@ -67,7 +67,6 @@ describe('PATCH /robots/{robotId}/robotName', () => {
       JSON.stringify('newTestRobot')
     );
 
-    // verify if really in DB
```
[the remark this answers is not in the document]
Comment (WolfgangDaniel) [reply server/api/routes/robots/robots.test.js:70]: I think this is clear even without the comment :)

File: server/api/routes/users/users.test.js [hunk 2]
```diff
@@ -43,8 +43,6 @@ describe('GET /users/{userId}/robots', () => {
     await ssotRetrievalController.getRobotList(request, response);
     const data = await response._getData();
     expect(response.statusCode).toBe(200);
-    // Catches error "Received: serializes to the same string"
-    // Solution found here https://github.com/facebook/jest/issues/8475#issuecomment-537830532
```
[the remark this answers is not in the document]
Comment (WolfgangDaniel) [reply server/api/routes/users/users.test.js]: Okay, fair point. I just put the comment back in.

File: server/api/routes/robots/robots.test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (SanJSp) [reply server/api/routes/robots/robots.test.js:70]: Ok, then I have nothing to add :)

File: frontend/src/components/App.jsx [hunk 3]
```diff
@@ -18,10 +18,10 @@ const App = () => (
       <Switch>
         <Route path='/' component={Home} exact />
         <Route path='/modeler/:robotId' component={RobotModeler} />
-        <Route path='/robotcode_editor' component={RobotFile} />
-        <Route path='/robot_overview' component={RobotOverview} />
+        <Route path='/robotcodeEditor' component={RobotFile} />
```
Comment (kej-jay) [inline frontend/src/components/App.jsx:21]:
```suggestion
        <Route path='/robotCodeEditor' component={RobotFile} />
```

File: server/socket/socketHelperFunctions.js [hunk 4]
```diff
@@ -56,7 +59,7 @@ exports.getRobotJobParameters = async (jobId) => {
 
 /**
  * @description Fetches the ssot from the database and parses the ssot to robot code
```
Comment (kej-jay) [inline server/socket/socketHelperFunctions.js:61]: I'm not sure, if this is correct

Comment (WolfgangDaniel) [reply server/socket/socketHelperFunctions.js]: ✅

**Review by kej-jay — APPROVED** [review]

**Pull request merged**